tableauserverclient/server/endpoint/workbooks_endpoint.py

```diff
@@ -141,6 +289,49 @@ def update(
         workbook_item: WorkbookItem,
         include_view_acceleration_status: bool = False,
     ) -> WorkbookItem:
+        """
+        Modifies an existing workbook. Use this method to change the owner or
+        the project that the workbook belongs to, or to change whether the
+        workbook shows views in tabs. The workbook item must include the
+        workbook ID and overrides the existing settings.
+
+        See https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_workbooks_and_views.htm#update_workbook
+        for a list of fields that can be updated.
+
+        Parameters
+        ----------
+        workbook_item : WorkbookItem
+            The workbook item to update. ID is required. Other fields are
+            optional. Any fields that are not specified will not be changed.
+
+        include_view_acceleration_status : bool, default False
+            Set to True to include the view acceleration status in the response.
+
+        Returns
+        -------
+        WorkbookItem
+            The updated workbook item.
+
+        Examples
+        --------
+
+        >>> import tableauserverclient as TSC
+        >>> tableau_auth = TSC.TableauAuth('username', 'password', site_id='site')
```

Comment:
I'm not sure about having all this sample code in the doc strings, it'll be easy to let it get out of date. I'm hoping to go the other direction and make it possible to run all the sample files regularly - perhaps significant code snippets could be put in a sample, and the comment could point there?

Reply:
I can remove the examples.